chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -63,6 +63,12 @@ if [ $(id -u) -gt 0 ]; then
     exit 1
 fi
 
+if test -n "$SSH_CONNECTION" -a -z "$STY"; then
```

Comment:
Yes, performance and behaviour are identical, I was talking purely about style.  However even though there was no predominant style within this particular file, the predominant style in the rest of SUSE Cloud is to use `[ ]`.  So we should try be consistent across the team and product.  Although on this occasion I didn't write "-1".